packages/babel-helper-skip-transparent-expr-wrappers/src/index.js

```diff
@@ -0,0 +1,32 @@
+import * as t from "@babel/types";
+import type { NodePath } from "@babel/traverse";
+
+export function getCallContext(callPath: NodePath): NodePath {
+  if (!callPath.isCallExpression() && !callPath.isOptionalCallExpression()) {
+    throw new Error(
+      `Expected type "CallExpression" or "OptionalCallExpression" ` +
+        `but instead got "${callPath.type}".`,
+    );
+  }
+
+  const calleePath = callPath.get("callee");
+  return skipTransparentExprWrappers(calleePath);
+}
+
+export function isTransparentExprWrapper(node: Node) {
```

Comment:
Can you add a comment to show the idea/concept of `TransparentExprWrapper`? Since it is not in the spec, the context of how this concept is come up will benefit new contributors.

Reply:
Good call, done!